lib/src/signed_video_h26x_common.c

```diff
@@ -271,18 +271,6 @@ set_hash_list_size(gop_info_t *gop_info, size_t hash_list_size)
   return SV_OK;
 }
 
-svrc_t
-reset_gop_hash(signed_video_t *self)
-{
-  if (!self) return SV_INVALID_PARAMETER;
-
-  gop_info_t *gop_info = self->gop_info;
-  assert(gop_info);
-
-  gop_info->num_nalus_in_gop_hash = 0;
-  return openssl_hash_data(self->crypto_handle, &gop_info->gop_hash_init, 1, gop_info->gop_hash);
```

Comment:
We should write a ticket to init the new gop hash computation with gop_hash_init.